less/index.less

```diff
@@ -1 +1,33 @@
-// Follow the order in the readme
+@import "variables.less";
+@import "navigation.less";
+@import "header.less";
+@import "section.less";
+@import "footer.less";
+
+hr {
+    border-top: dashed 1.5px lightgray;
+}
+
+h1, h2, h3, h4 {
+    font-family: 'Indie Flower';
+    font-weight: normal;
+    color: black;
+}
+
+a {
+    color: inherit;
+    text-decoration: none;
+}
+
+@media (min-width: @desktop-min-width) {
+    .container {
+        width: 50%;
```

Comment:
The max-width of your container should be 800px to max the design file, not 50%.